contracts/BC_fusion/GovToken.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/ERC20BurnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/ERC20PermitUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/ERC20VotesUpgradeable.sol";
+
+import "./System.sol";
+import "./interface/IStakeCredit.sol";
+
+contract GovToken is
+    System,
+    Initializable,
+    ERC20Upgradeable,
+    ERC20BurnableUpgradeable,
+    ERC20PermitUpgradeable,
+    ERC20VotesUpgradeable
+{
+    string private constant NAME = "BSC Governance Token";
+    string private constant SYMBOL = "govBNB";
+
+    // validator StakeCredit contract => user => amount
+    mapping(address => mapping(address => uint256)) public mintedMap;
+
+    function initialize() public initializer onlyCoinbase onlyZeroGasPrice {
+        __ERC20_init(NAME, SYMBOL);
+        __ERC20Burnable_init();
+        __ERC20Permit_init(NAME);
+        __ERC20Votes_init();
+    }
+
+    function sync(address[] calldata validatorPools, address account) external onlyStakeHub {
```

Comment:
Please dont name as `validatorPool`, change to `stakeCredit`

Reply:
fixed